engines/sword1/resman.cpp

```diff
@@ -496,6 +499,31 @@ void ResMan::openScriptResourceLittleEndian(uint32 id) {
 	}
 }
 
+uint32 ResMan::getDeathFontId() {
+	// At some point in the releases (as evidenced by the disasms of all
+	// known executables, and by the source code in our possession), Revolution
+	// changed the resource offsets for some files. I have tried EVERYTHING to
+	// try and discern which file we are dealing with and spectacularly failed.
+	// The only choice which seems to work correctly is to check the file size,
+	// as the newer GENERAL.CLU file is bigger. Sorry.
```

Comment:
That is a valid approach, especially taking into account that practically all the official version releases are now known

Reply:
Thank you, if it works for you then it works for me 🙂